SPECS/SymCrypt/SymCrypt.spec

```diff
@@ -1,15 +1,17 @@
-%define debug_package %{nil}
 Summary:        A core cryptographic library written by Microsoft
 Name:           SymCrypt
 Version:        103.4.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System/Libraries
 URL:            https://github.com/microsoft/SymCrypt
 Source0:        https://github.com/microsoft/SymCrypt/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Source1:        https://github.com/smuellerDD/jitterentropy-library/archive/v3.3.1.tar.gz#/jitterentropy-library-3.3.1.tar.gz
+Source2:        find-debuginfo
```

Comment:
Where did we get the modified `find-debuginfo` script from? If it's an updated version of the one from the `debugedit` package, then I think it might even better to update that script in `debugedit.spec` itself. This way it's available everywhere.

Reply:
I feel pretty strongly that we should *not* do that -- this is currently `SymCrypt`-specific and we don't want to potentially cause problems elsewhere.

If at some point `eu-strip` supported `--keep-symbol`, I'd be all for piping that argument through `find-debuginfo` everywhere, but that isn't true right now.